src/main/controllers/NoticePayController.ts

```diff
@@ -11,7 +12,92 @@ import { assignFormData, getPageContent, handleSessionErrors, setUserCase } from
 export default class NoticePayController {
   private readonly form: Form;
   private readonly noticePayContent: FormContent = {
-    fields: {},
+    fields: {
+      noticePeriodLength: {
+        id: 'notice-period-length',
+        type: 'radios',
+        classes: 'govuk-radios',
+        values: [
+          {
+            label: (l: AnyRecord): string => l.yes,
+            value: YesOrNo.YES,
+            selected: false,
+            subFields: {
+              noticePeriodUnit: {
+                id: 'notice-period-unit',
+                type: 'option',
+                classes: 'govuk-radios--inline',
```

Comment:
If you look under the definitions folder and the radios.ts file, you should see some code there which you can resuse/extend for radio buttons